src/apps/dictionaries/components/ReleasedVersions.tsx

```diff
@@ -105,7 +105,7 @@ const ReleasedVersions: React.FC<Props> = ({
                 {versionsToDisplay.map((row: APIDictionaryVersion) => (
                   <TableRow key={row.id}>
                     <TableCell>{row.id}</TableCell>
-                    <TableCell>{row.description || "None"}</TableCell>
+                    <TableCell style={{wordBreak: 'break-all'}}>{row.description || "None"}</TableCell>
```

Comment:
Don't we need the same style for row.id as well? version is also can take a large string value.

Reply:
@kirity-mahanty will add

Reply:
thanks.